tests/test_black.py

```diff
@@ -1289,6 +1318,195 @@ def test_shhh_click(self) -> None:
             except RuntimeError as re:
                 self.fail(f"`patch_click()` failed, exception still raised: {re}")
 
+    @async_test
+    async def test_blackd_request_needs_formatting(self) -> None:
+        if not has_blackd_deps:
+            self.skipTest("blackd's dependencies are not installed")
+            return
```

Comment:
unittest has this decorator called @unittest.skipIf that you could use here instead of this.